pkg/odo/cli/url/create.go

```diff
@@ -58,42 +60,65 @@ func (o *URLCreateOptions) Complete(name string, cmd *cobra.Command, args []stri
 	} else {
 		o.Context = genericclioptions.NewContext(cmd)
 	}
+	if experimental.IsExperimentalModeEnabled() {
+		if len(args) == 0 {
+			o.urlName = url.GetURLName(o.Component(), o.urlPort)
+		} else {
+			o.urlName = args[0]
+		}
+		err = o.InitEnvInfoFromContext()
```

Comment:
Should we be checking whether `err != nil` here and return err?

Reply:
updated